tools/cpp/windows_cc_configure.bzl

```diff
@@ -786,11 +792,13 @@ def configure_windows_toolchain(repository_ctx):
     template_vars = dict()
     msvc_vars_x64 = _get_msvc_vars(repository_ctx, paths, "x64")
     template_vars.update(msvc_vars_x64)
-    template_vars.update(_get_clang_cl_vars(repository_ctx, paths, msvc_vars_x64))
+    template_vars.update(_get_clang_cl_vars(repository_ctx, paths, msvc_vars_x64, "x64"))
     template_vars.update(_get_msys_mingw_vars(repository_ctx))
     template_vars.update(_get_msvc_vars(repository_ctx, paths, "x86", msvc_vars_x64))
     template_vars.update(_get_msvc_vars(repository_ctx, paths, "arm", msvc_vars_x64))
-    template_vars.update(_get_msvc_vars(repository_ctx, paths, "arm64", msvc_vars_x64))
+    msvc_vars_arm64 = _get_msvc_vars(repository_ctx, paths, "arm64", msvc_vars_x64)
```

Comment:
nit, just:
```
    template_vars.update(_get_msvc_vars(repository_ctx, paths, "arm64", msvc_vars_x64))
```
?

Reply:
`msvc_vars_arm64` is required later (line 801) so have to store it in a variable

Reply:
Got it, thanks!